ui/components/multichain/account-list-item/account-list-item.types.js

```diff
@@ -0,0 +1,5 @@
+export const MenuOptionTypes = {
+  None: 'none',
+  ConnectionMenu: 'connectionMenu',
```

Comment:
I don't think we need `Menu` in these keys because `Menu` is in the object name.  This would be an improvement:

```
{
  None: 'none',
  Connection: 'connection',
  Account: 'account'
}
```

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/23232/commits/f04561a22fb16e2948e6b45218a8857144784531